core/src/main/java/com/amplifyframework/core/Amplify.java

```diff
@@ -192,11 +191,7 @@ private static void beginInitialization(@NonNull Category<? extends Plugin<?>> c
 
         synchronized (CONFIGURATION_LOCK) {
             if (CONFIGURATION_LOCK.get()) {
-                final String updateString = registryUpdateType.name().toLowerCase(Locale.US);
-                throw new AmplifyException(
-                    "The client tried to " + updateString + " a plugin after calling configure().",
-                        "Plugins may not be added or removed after configure(...) is called."
-                );
+                throw new AlreadyConfiguredException();
```

Comment:
The recovery suggestion ("Remove the duplicate call to `Amplify.configure()`") doesn't make sense for this scenario.  Can you update `AlreadyConfiguredException` to take a recovery suggestion as a constructor parameter?    Then, pass a custom recovery suggestion like this:

```suggestion
                throw new AlreadyConfiguredException("Plugins may not be added or removed after configure(...) is called.");
```

Reply:
@richardmcclellan Done!